certbot/constants.py

```diff
@@ -95,3 +95,6 @@
 
 RENEWER_CONFIG_FILENAME = "renewer.conf"
 """Renewer config file name (relative to `IConfig.config_dir`)."""
+
+FORCE_INTERACTIVE_FLAG = "--force-interactive"
```

Comment:
Yeah not using a literal here is probably fine because we shouldn't be changing CLI flags on people, but this only adds one line of code and adds a bit of extra flexibility. I'd prefer to leave this as is, but I can change it if you really want it to be consistent with the rest of the code.